app/components/forms/events/view/create-access-code.js

```diff
@@ -0,0 +1,111 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+import $ from 'jquery';
+
+const { Component, computed } = Ember;
+
+export default Component.extend(FormMixin, {
+  getValidationRules() {
+    $.fn.form.settings.rules.checkMaxMin = () => {
+      if (this.get('data.response.minQuantity') > this.get('data.response.maxQuantity')) {
+        return false;
+      }
+      if (this.get('data.response.maxQuantity') > this.get('ticketsNumber')) {
+        return false;
+      }
+      return true;
+    };
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+
+      fields: {
+        accessCode: {
+          identifier : 'access_code',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter access code')
+            },
+            {
+              type  : 'regExp',
+              value : '^[a-zA-Z0-9_-]*$'
+            }
+          ]
+        },
+        numberOfAccessTickets: {
+          identifier : 'number_of_access_tickets',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter number of tickets')
+            },
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Please enter proper number of tickets')
+            }
+          ]
+        },
+        allTicketTypes: {
+          identifier : 'all_ticket_types',
+          rules      : [
+            {
+              type   : 'checked',
+              prompt : this.l10n.t('Please select the appropriate choices')
+            }
+          ]
+        },
+        min: {
+          identifier : 'min',
+          optional   : true,
+          rules      : [
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Please enter the proper number')
+            },
+            {
+              type   : 'checkMaxMin',
+              prompt : this.l10n.t('Please check the values. Minimum value might be greater than maximum or maximum value might be greater than total tickets.')
```

Comment:
The error message should be only about this field. In this case, it should only say,

`Minimum value should not be greater than maximum`